idris.cabal

```diff
@@ -108,12 +108,12 @@ custom-setup
 
 Flag FFI
   Description:  Build support for libffi
-  Default:      False
```

Comment:
These flags should stay as they are, because libffi can be difficult for Mac users and some people don't like the license of GMP.

Reply:
And I agree, I'm sort of asking how should this be handled in the second bullet point.

Reply:
I think that their behavior should remain unchanged in the .cabal file at least. Nix users are a minority of Idris users, and I don't think that we should make it more difficult to build it the common way to make Nix packaging easier.